.buildkite/hooks/pre-command

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# On some distros, this directory ends up not readable by the `elasticsearch` user that gets created during tests
+# This fixes that
+chmod 755 ~
+
+WORKSPACE="$(pwd)"
+export WORKSPACE
```

Comment:
Is there a reason not to just combine these lines into something like `export WORKSPACE="$(pwd)"`?

Reply:
I generally follow `shellcheck` guidelines when writing shell, and it's one of them: https://www.shellcheck.net/wiki/SC2155